setup.cfg

```diff
@@ -31,7 +31,7 @@ universal = 1
 universal = 1
 
 [flake8]
-ignore = D100,D101,D102,D103,D104,D105,D106,D107
+ignore = D100,D101,D102,D103,D104,D105,D106,D107,W504
```

Comment:
I'd prefer if that was ignored on a per line basis.

Reply:
Ok. Curious how black would split up those lines.